test/EncumberableToken.t.sol

```diff
@@ -196,6 +196,35 @@ contract EncumberableTokenTest is Test {
         assertEq(wrappedToken.balanceOf(charlie), 40e18);
     }
 
+    function testTransferFromRevertsIfYouSpendEncumberedTokens() public {
```

Comment:
nit: `testTransferFromRevertsIfSpendingTokensEncumberedToOthers`

Reply:
I like it; updated